scripts/fetch-external-resources

```diff
@@ -33,7 +33,7 @@ RESOURCES = [
     'content/_tmp/examples'
   ],
   [
-    'https://ci.jenkins.io/job/Infra/job/pipeline-steps-doc-generator/job/master/lastSuccessfulBuild/artifact/allAscii.zip',
+    'https://reports.jenkins.io/allAscii.zip',
```

Comment:
Nit: A more descriptive file name might be better when it's a standalone file without context provided through the rest of the URL.

Reply:
Would a comment pointing to the source and an explanation be ok?
Asking because even with the URL the context was already hard to catch (unless you're the author of the whole chain).